src/index.d.ts

```diff
@@ -275,6 +275,11 @@ export namespace h {
 // Preact render
 // -----------------------------------
 
+export function render(
+	vnode: ComponentChild,
+	parent: Element | Document | ShadowRoot | DocumentFragment,
+): void;
+/** @deprecated Will be removed in v11. */
```

Comment:
Unsure of what we want to suggest in the message. Maybe link to https://gist.github.com/developit/f4c67a2ede71dc2fab7f357f39cff28c until [better docs are written](https://github.com/preactjs/preact/issues/3542#issuecomment-1145445352)?

Reply:
That would be good, yep.